utils/statediff/src/main.rs

```diff
@@ -26,31 +28,14 @@ impl ChainCommand {
     pub async fn run(&self) {
         let dir = ProjectDirs::from("com", "ChainSafe", "Forest").unwrap();
         let chain_path = dir.data_dir().join(&self.chain);
-        let blockstore = open_db(&chain_path);
+        let blockstore = open_db(&db_path(&chain_path), &DbConfig::default()).unwrap();
```

Comment:
Let's replace the `unwrap`s  with proper error handling.

Reply:
Fixed